packages/ui/src/components/ConfirmModal/index.tsx

```diff
@@ -37,6 +37,14 @@ const ConfirmModal = ({
       overlayClassName={classNames(modalStyles.overlay, styles.overlay)}
       parentSelector={() => document.querySelector('main') ?? document.body}
       ariaHideApp={false}
+      onAfterOpen={() => {
+        /* eslint-disable @silverhand/fp/no-mutation */
+        document.body.style.overflow = 'hidden';
+      }}
+      onAfterClose={() => {
+        document.body.style.overflow = 'unset';
+        /* eslint-enable @silverhand/fp/no-mutation */
+      }}
```

Comment:
nit: if not use css, maybe better
```suggestion
      /* eslint-disable @silverhand/fp/no-mutation */
      onAfterOpen={() => {
        document.body.style.overflow = 'hidden';
      }}
      onAfterClose={() => {
        document.body.style.overflow = 'unset';
      }}
      /* eslint-enable @silverhand/fp/no-mutation */
```